doc/sphinx/source/recipes/recipe_sanity_checks.rst

```diff
@@ -0,0 +1,75 @@
+.. _recipe_santiy_checks:
+
+Sanity checks
+=============
+
+Overview
+--------
+
+The sanity check recipe (recipe_monitor_sanity_checks.yml) is intended to perform
+some basic sanity checks for new simulations during model development. A subset of variables
+is selected to check whether the model is performing in reasonable way. The recipe plots global
+monthly means as well as maximum and minimum across all individual grid cells for each time step.
+This can be used to check for example for negative mass concentrations occuring at any grid cell
+at any time (minimum is less than zero) or whether individual grid cells exceed physically reasonalble
+values (e.g. total cloud fraction greater than 100%).
+The global monthly means can be compared to the minimum and maximum values found in reference
+datasets (observations, reanalyses) across all months and all reference datasets. The "reasonable"
+limits are shown as red lines. A good model simulations would be expected to not exceed these
+limits at any time. The minimum and maximum global averages across all months and all reference
+datasets is calculated with recipe_create_ranges_obs.yml
+
+Available recipes and diagnostics
```

Comment:
You can also add the diagnostic here, similar to the model evaluation doc: https://github.com/ESMValGroup/ESMValTool/blob/main/doc/sphinx/source/recipes/recipe_model_evaluation.rst

Reply:
Done